package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "kakunin-cli",
+  "version": "1.0.0",
+  "description": "Command-line interface for Kakunin E2E framework",
+  "homepage": "https://thesoftwarehouse.github.io/Kakunin/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/TheSoftwareHouse/kakunin-cli.git"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "The Software House",
+    "url": "http://tsh.io"
+  },
+  "contributors": [
+    {
+      "name": "Józef Szymala"
+    }
+  ],
+  "main": "./dist/index.js",
+  "bin": {
+    "kakunin-cli": "./dist/src/index.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "lint": "tslint -c tslint.json 'src/**/*.ts'",
+    "prepublishOnly": "npm run build"
```

Comment:
npm run lint && npm run build

so it will fail if some linting errors occured

Reply:
Updated: `"prepublishOnly": "npm run lint && npm run build"`